### PR TITLE
Respond ember routing diagnostic

### PR DESCRIPTION
**Pull request by MichaelB167**

### CODE REVIEW

**Comment by MichaelB167** [comment]
Comfort: 3
Clarity: 4

I used my notes, the ember routing training repos and google

File: diagnostic.md [hunk 1]
```diff
@@ -6,21 +6,21 @@ Record your responses inside the fenced code blocks below each question.
...
+    ember g route campus/boston
     ```
 
 1.  Suppose you have a nested route at the URL `/campus/boston`. How would you
     use the `link-to` helper to generate an appropriate link?
 
     ```md
-    <!-- your response here -->
+    {{#link-to '/campus/boston'}}Boston{{/link-to}}
     ```
```
Comment (laurenfazah) [inline diagnostic.md:24]: Close! `{{#link-to 'campus.boston'}}Boston{{/link-to}}`

File: diagnostic.md [hunk 2]
```diff
@@ -35,7 +35,9 @@ Record your responses inside the fenced code blocks below each question.
...
 
     ```md
-    <!-- your response here -->
+    The second route is dynamic and nests the product id within products.
+    The second route is an example of a masking path.
+    The second route represents a different view state.
     ```
```
Comment (laurenfazah) [inline diagnostic.md:41]: Great, and for more context:

The first `'product'` route definition is nested under the `'products'`
definition, whereas the second `'product'` route definition is a sibling of
the `'products'` definition.

Nested route definitions, like the first `'product'` route above, require
a nested file structure, such that Route and Template files for `'product'`
are expected to be found at `app/products/product`. For sibling routes,
folders are siblings as well.

The first `'product'` route definition instructs Ember to render
`app/products/product/template.hbs` in the context (the `outlet`) of the
parent `app/products/template.hbs`. The second `'product'` route definition
instructs Ember to render `app/product/template.hbs` in the context of the
application template.

These two route definitions do **not** have different paths.

File: diagnostic.md [hunk 3]
```diff
@@ -48,11 +50,23 @@ Record your responses inside the fenced code blocks below each question.
...
+    export default Ember.Route.extend({
+      model: function(){
+        return [
+          {
+          id: 123
+          },
+        ]
+      }
+    });
     ```
```
Comment (laurenfazah) [inline diagnostic.md:66]: So actually here, we'd want to take advantage of the fact that a Route model can accept a parameter of `params` that gives us access to all route params. Thus, in the model, we'd access the value 123 coming from the route params with `params.movie_id`

**Comment by laurenfazah** [comment]
Great work, Michael! 4/6

**Pull request closed without merging**